src/Vehicle/Vehicle.cc

```diff
@@ -199,6 +201,17 @@ Vehicle::Vehicle(LinkInterface*             link,
 
     _vehicleLinkManager->_addLink(link);
 
+    QObject::connect(_vehicleLinkManager, &VehicleLinkManager::communicationLostChanged, this, [](bool is_lost) {
+        if(is_lost && !_softrebootRequested) { // supposedly here this is not a intentional reboot - this trusts is NOT emitted on reboot....
```

Comment:
All logic related to _softRebootRequested  seems to be redundant. It will never be true here, as the communicationLostChanged-signal is not emitted on soft reboot